scvi/inference/trainer.py

```diff
@@ -42,8 +42,11 @@ class Trainer:
     default_metrics_to_monitor = []
 
     def __init__(self, model, gene_dataset, use_cuda=True, metrics_to_monitor=None, benchmark=False,
-                 verbose=False, frequency=None, weight_decay=1e-6, early_stopping_kwargs=dict(),
-                 data_loader_kwargs=dict(), show_progbar=True):
+                 verbose=False, frequency=None, weight_decay=0, early_stopping_kwargs=None,
```

Comment:
It looks like you're changing `weight_decay` here from 1e-6 to 0. It should be the former I think, right?